.github/workflows/ansible-test.yml

```diff
@@ -100,67 +85,84 @@ jobs:
     needs: [build]
     strategy:
       matrix:
-        python: ['3.8', '3.9', '3.10']
+        python: ['3.8', '3.9', '3.10', '3.11']
         ansible:
           - stable-2.13
           - stable-2.14
           - stable-2.15
           - devel
         exclude:
+          - ansible: stable-2.13
+            python: '3.11'
           - ansible: stable-2.14
             python: '3.8'
           - ansible: stable-2.15
             python: '3.8'
+          - ansible: devel
+            python: '3.8'
+          - ansible: devel
+            python: '3.9'
     runs-on: ubuntu-latest
     steps:
       - name: Perform sanity testing
         uses: ansible-community/ansible-test-gh-action@release/v1
         with:
           ansible-core-version: ${{ matrix.ansible }}
-          target-python-version:  ${{ matrix.python }}
+          target-python-version: ${{ matrix.python }}
           testing-type: sanity
           pull-request-change-detection: true
+          coverage: never
 
   lint:
-      name: Ansible lint
-      runs-on: ubuntu-latest
-      needs: [build]
-      strategy:
-        fail-fast: false
-        matrix:
-          python-version: [3.9, '3.10']
-          ansible-version: [stable-2.12, stable-2.13, stable-2.14, stable-2.15]
-
-      steps:
-        # Important: This sets up your GITHUB_WORKSPACE environment variable
-        - name: Checkout the source code
-          uses: actions/checkout@v3
-          with:
-            fetch-depth: 0 # needed for progressive mode to work
-        
-        - name: Set up Python ${{ matrix.python-version }}
-          uses: actions/setup-python@v1
-          with:
-            python-version: ${{ matrix.python-version }}
-
-        - name: Install ansible (${{ matrix.ansible-version }}) version
-          run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible-version }}.tar.gz --disable-pip-version-check
-
-        - name: Install ansible lint
-          run: pip install ansible-lint==6.16.2 pytest --disable-pip-version-check
-
-        - name: Install ansible posix
-          run: ansible-galaxy collection install ansible.posix
-
-        - name: Download migrated collection artifacts
-          uses: actions/download-artifact@v1
-          with:
-            name: collection
-            path: .cache/collection-tarballs
-
-        - name: Install collection build
-          run: ansible-galaxy collection install .cache/collection-tarballs/*.tar.gz
-
-        - name: Run Ansible lint
-          run: ansible-lint --show-relpath
-          working-directory: /home/runner/work/dellemc-openmanage-ansible-modules/dellemc-openmanage-ansible-modules
+    name: Ansible lint
+    runs-on: ubuntu-latest
+    needs: [build]
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.9', '3.10', '3.11']
+        ansible-version: [stable-2.12, stable-2.13, stable-2.14, stable-2.15, devel]
```

Comment:
ansible stable-2.12 - Is it required for lint?

Reply:
Done